fulp_modules/features/nanites/code/programs/utility.dm

```diff
@@ -177,8 +177,8 @@
 
 /datum/nanite_program/spreading/active_effect()
 	if(spread_cooldown < world.time)
+		spread_cooldown = world.time + 50
```

Comment:
Asking again, isn’t this gonna just remove the cooldown, since you’re only setting it if it’s already on cooldown?

Reply:
This is true, reverting to the original and flipping the sign should fix this